src/index-hindi.html

```diff
@@ -0,0 +1,655 @@
+    .hidden { display: none; }
+    .question { margin: 2em 0 0 0; }
+    .answer {
+        margin: 1em 0;
+        padding: 0 1em;
+        border: none;
+        border-radius: 4px;
+        max-height: 150px;
+        overflow: hidden;
+        box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5);
+        transition: 200ms all ease-in;
+    }
+    .button-bar .btn { margin-bottom: 5px }
+
+    .answer.focused {
+        overflow: auto;
+        transition: 200ms all ease-in;
+        outline: none;
+    }
+
+    #copy-instruction { display: none; }
+    #response-copied { display: none; }
+    #responseTextarea { width: 100%; height: 10em; font-size: 15px; line-height: 1em; border: 1px solid #ccc; }
+    label[for=responseSubject] {width: 100%;}
+    #responseSubject {width: 100%; padding: 2px;}
+    .social-col {
+      display: inline-block;
+      padding: 0 8px;
+      vertical-align: top;
+    }
+    #report-problem {
+      font-weight: bold;
+      position: fixed;
+      right: -4px;
+      opacity: 0.9;
+      bottom: -4px;
+    }
+  </style>
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+  ga('create', 'UA-9752294-8', 'auto');
+  ga('set', 'forceSSL', true);
+  ga('require', 'linkid');
+  ga('send', 'pageview');
+</script>
+<script>
+    var shiftWindow = function() { scrollBy(0, -50) };
+    window.addEventListener("hashchange", shiftWindow);
+    if (window.location.hash) shiftWindow();
+</script>
+</head>
+<body>
+  <nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#">#SaveTheInternet</a>
+        <a class="navbar-toggle btn btn-info hidden-md hidden-lg navBarSend sendResponse disabled" style="display:none;" id="mobileNavBarSend">Send</a>
+      </div>
+      <div class="nav navbar-nav navbar-right hidden-xs hidden-sm navBarSend" style="display:none;">
+        <button type="button" class="btn btn-info navbar-btn sendResponse disabled" id="navBarSend">Respond to TRAI now</button>
+      </div>
+    </div>
+  </nav>
+  <div id="header">
+  </div>
+  <div class="container">
+    <div class="row">
+         <div class="col-xs-12 col-sm-6 col-md-9 col-lg-9">
+
+            <h1> इंटरनेट को बचायें (भाग 3)।
+            एक बार फिर से हमें TRAI <small>(भारतीय दूरसंचार नियामक प्राधिकरण)</small> को बताने की जरूरत है की हम इंटरनेट की निष्पक्षता के पक्ष में हैं (दिसंबर 2015)
+    </h1>
+
+
+            <div class="alert alert-warning hidden" id="browser_warning" role="alert">
+              <span class="browser_name">This browser</span> is known to have problems submitting this form. Please use another browser if it doesn't work.
```

Comment:
We should translate this too, for people using browsers we have known problems with.